report/main.qmd

```diff
@@ -0,0 +1,106 @@
+
+Hence, the proposed models are \begin{align*}
+ZeroCal & \sim Intervention(Site) + DofW + Total\\
+Sugary & \sim Intervention(Site) + DofW + Total.
+\end{align*} Fixed effects include Intervention, DofW, and Total, while the random effect measures how the effect of Intervention and baseline consumption varies across sites. Note that these models are simplified versions, especially with the omission of the link function. Please refer to the appendix for the formulated models.
+
+However, there are several limitations to the models mentioned above. Firstly, GLLMs do not accommodate autocorrelation in time series data, which refers to the dependence between consecutive observations. Moreover, the model proposed implicitly assumes that the response variables follow Poisson distributions [@coxe2009analysis], where the expectation and variability of response variables should equate, but this assumption can be easily violated in practice.
+
+## Conclusions
+
+We recommend using GLMMs to fit the relationship between covariates and response variables. Based on different research interests, response variable should be either ZeroCal beverage consumption or sugary beverage consumption. The fixed-effect explanatory variables suggested are intervention, Total and DofW, and the random-effect explanatory variable is Site.
+
+## References
+
+::: {#refs}
+:::
+
+## Statistical Appendix
+
+### Mathematical Formulation of Models
+
+Here is a formulation of the Generalized Linear Mixed Model within the context of this research. As suggested earlier, random effects can be placed on both the intervention's slope and intercept. Taking $response = ZeroCal$ as an example, the model will take the format of $$
+E(zero\_calorie_{ij}) = exp((\beta_0 + b_{0i}) + (\beta_1 + b_{1i})\; Intervention + \beta_2 \; DofW + \beta_3 \; Total)
```

Comment:
Again, no text, especially with underscores, should be used as variables in math equations.